### PR TITLE
CHAT-390: User's display name should be grey colored in discussion

### PR DESCRIPTION
**Pull request by hassenelkamel-exo**
In the case of `disabled user`, user’s display should be grey colored in discussion-block. so we get the status of disabled from message and  add a `Css` class to apply this style in it.

### CODE REVIEW

File: application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue [hunk 1]
```diff
@@ -204,6 +204,10 @@ export default {
       confirmMessage: '',
       confirmOKMessage: '',
       confirmKOMessage: '',
+      statusMap : {
+        inactive: this.$t('exoplatform.chat.inactive'),
+        available: this.$t('exoplatform.chat.available')
```
Comment (ahamdi) [inline application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue]: Is this used ?

File: application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue [hunk 2]
```diff
@@ -264,6 +268,20 @@ export default {
     },
     specificMessageClass() {
       return this.specificMessageObj ? this.specificMessageObj.iconClass : '';
+    },
+    statusStyle: function() {
+      if (this.message.isEnabledUser === 'false' && !this.isCurrentUser) {
+        return 'user-disabled ';
```
Comment (ahamdi) [inline application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue]:
```suggestion
        return 'user-disabled';
```

File: application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue [hunk 3]
```diff
@@ -264,6 +268,20 @@ export default {
...
+      }
+    },
+    getStatus() {
+      const paraOpen='(';
+      const paraClose=')';
```
Comment (ahamdi) [inline application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue]: No need for both variables paraOpen and paraClose . return directly the translated value of disabled i.e this.$t('exoplatform.chat.inactive')

Comment (hassenelkamel-exo) [reply application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue]: ok done

File: application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue (hunk 2 above)
Comment (hassenelkamel-exo) [reply application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue]: ok done

File: application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue [hunk 4]
```diff
@@ -264,6 +264,16 @@ export default {
     },
     specificMessageClass() {
       return this.specificMessageObj ? this.specificMessageObj.iconClass : '';
+    },
+    statusStyle: function() {
+      if (this.message.isEnabledUser === 'false' && !this.isCurrentUser) {
+        return 'user-disabled';
+      }
+    },
+    getStatus() {
```
Comment (ahamdi) [inline application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue]:
```suggestion
    disabledStatus() {
```

File: application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue [hunk 5]
```diff
@@ -6,7 +6,7 @@
     </div>
     <div v-hold-tap="openMessageActions" class="chat-message-bubble">
       <div v-if="displayUserInformation && (!message.isEnabledUser || message.isEnabledUser === 'true')" class="sender-name"><a :href="getProfileLink(message.user)">{{ message.fullname }}</a> :</div>
-      <div v-else-if="displayUserInformation" class="sender-name">{{ message.fullname }} :</div>
+      <div v-else-if="displayUserInformation" :class="statusStyle" class="sender-name">{{ message.fullname }} {{ getStatus }}: </div>
```
Comment (ahamdi) [inline application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue]:
```suggestion
      <div v-else-if="displayUserInformation" :class="statusStyle" class="sender-name">{{ message.fullname }} {{ disabledStatus }}: </div>
```

File: application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue (hunk 4 above)
Comment (hassenelkamel-exo) [reply application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue]: ok done

File: application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue (hunk 5 above)
Comment (hassenelkamel-exo) [reply application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue]: ok done

**Pull request merged**